contributing/labeling.md

```diff
@@ -0,0 +1,15 @@
+# Label Definitions
+
+* `Epic`: Big issue to be broken down into smaller tasks
+
+### Priority (`p.*`)
+* `p.Critical`: To be handled as soon as possible, usually a bug or security loophole
+* `p.High`: Issues corresponding to the upcoming `Milestone`
+* `p.Medium`: Issues corresponding to a future `Milestone`
+* `p.Low`: Good-to-have features not in the roadmap
+
+### Difficulty (`d.*`)
```

Comment:
What do you think of leaving Advanced for issues that require more indepth research? We can leave normal issues untagged in this case.